extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -71,6 +71,12 @@ export class BaseCarousel extends AMP.BaseElement {
     // TODO(erwinm): Does label need i18n support in the future? or provide
     // a way to be overridden.
     this.prevButton_.setAttribute('aria-label', 'Previous item in carousel');
+    this.prevButton_.setAttribute('tabindex', 0);
+    this.prevButton_.onkeydown = event => {
+      if (event.keyCode == 13 /* enter */ || event.keyCode == 32 /* space */) {
```

Comment:
one nit: instead if adding comments , use enums 
so you can have consts like KEYCODE.ENTER/ KEYCODE.SPACE will make your code readable.